### ISSUE TITLE
Can't get @glide:mount-after to work

### ISSUE THREAD

**Issue by drehimself** [antonreshetov/vue-glide#27]
Hello,

I'm trying to use the mount-after event but it doesn't seem to fire. I'm not sure what I'm missing. Thank you!

https://codesandbox.io/s/86ok92oy9

```js
<vue-glide
  class="demo"
  ref="slider"
  v-model="active"
  type="carousel"
  @glide:slide-click="test"
  @gilde:mount-after="testMount"
>
```

```js
methods: {
  test(index) {
    alert(`click ${index}`);
  },
  testMount() {
    alert("mounted");
  }
}
```

**Comment by stasoft91** [comment]
+1

**Comment by antonreshetov** [comment]
Hi @drehimself 

The point is that glide.js is mounted immediately after the creation of the vue component `vue-glide`.
https://github.com/antonreshetov/vue-glide/blob/master/src/components/Glide.js#L234
Of course, the mount can be brought into the property of the initialization of the mount `<vue-glide: mount = "true" />`, but it seems to me that it is not entirely advisable

**Comment by antonreshetov** [comment]
Hi @drehimself 

1. No correct event in your code`@gilde:mount-after` => `@glide:mount-after`
2. Fixed in 8349e40d393f10a4baf1f7dd457907c00d73502c
